src/app/main/component/eco-news/components/create-edit-news/create-edit-news.component.ts

```diff
@@ -104,6 +104,11 @@ export class CreateEditNewsComponent extends FormBaseComponent implements OnInit
   }
 
   private getAllTags() {
+    if (localStorage.getItem('newsTags')) {
```

Comment:
we should use localStorageSerrvice for this

Reply:
done